src/wp-content/themes/twentyeleven/functions.php

```diff
@@ -79,8 +79,14 @@ function twentyeleven_setup() {
 		 * If you're building a theme based on Twenty Eleven, use
 		 * a find and replace to change 'twentyeleven' to the name
 		 * of your theme in all the template files.
+		 *
+		 * Manual loading of text domain is not required after the introduction of
+		 * just in time translation loading in WordPress version 4.6.
+		 * @ticket 58318
```

Comment:
```suggestion
		 * just in time translation loading in WordPress version 4.6.
		 *
		 * @ticket 58318
```

Add one line break. Apply similar to all other instance.

Reply:
@mukeshpanchal27 I have made the required changes.